### PR TITLE
Cell

### PR DESCRIPTION
**Pull request by cunninghamge**
I was able to make the render method a bit neater by taking advantage of the fact that once an iteration reaches a boolean equal to true it will carry out the action and not continue testing any of the subsequent cases. If a cell is empty it can only render "M" or ".", and after those are ruled out the cell must not be empty so the !empty is unnecessary for each of the following arguments. I'm open to adding it back in for readability but we don't need it to make the code work (and I happen to think the code looks very pretty without it). Since the code block follows the process of elimination we don't actually even need the render_ships argument for the iteration, although we do need the optional argument placeholder when we call the method. All tests are passing.

### CODE REVIEW

File: lib/cell.rb [hunk 1]
```diff
@@ -26,23 +26,16 @@ def fire_upon
   end
 
   def render(render_ships = false)
-    if fired_upon? && empty?
+    if empty? && fired_upon?
       "M"
-    elsif !fired_upon? && empty?
+    elsif empty? && !fired_upon?
       "."
```
Comment (GrayMyers) [inline lib/cell.rb:32]: Good thinking!  This is definitely an improvement over what we had before. One adjustment that you could make to simplify it further, now that the logic is a lot more readable, would be to move the fired_upon checks into the if statement for empty, which would make it a little easier to understand.

Comment (cunninghamge) [reply lib/cell.rb:32]: Do you mean like creating a nested if statement, with if empty? and then if fired_upon?

I guess we could also just do if empty? && fired_upon?, with the next test being elsif empty? (and the !fired_upon will be implicit).
